src/SimpleRSS.Reader/Paging.fs

```diff
@@ -0,0 +1,39 @@
+namespace SimpleRSS.Reader
+
+module Paging =
+    type PagingOptions =
+        {
+          /// <summary>The amount of items to include on each page.</summary>
+          pageSize: int
```

Comment:
You could be more specific here, you've got `int` everywhere, I would do something like create a domain specfic type, so you have the `PagingOptions` created by a function, something like:

```fs
let createPagingOptions size number = 
    if size > 0 && number > 0
    then Some ( { pageSize = size; pageNumber = number} )
    else None
```

Using an option but I would could also create a custom type - something like `ValidatedPagingOptions` and that should be what the function expects

The idea is to ensure that the consumer can't pass invalid data in the first place, so validating before making the call to the lib instead of after

Reply:
I thought about how strictly the type should enforce rules, and I ended up going for the quickest solution. However, I'm with you that it's beneficial to be stricter with it. I'll refactor the paging options a bit to be safer.

Reply:
Fixed in a8165147e879b0feacb0b01fac8e76790ea3b248.